src/main.rs

```diff
@@ -185,3 +196,53 @@ fn parse_args() -> Result<LndCfg, ArgsError> {
 
     Ok(LndCfg::new(address, cert_file, macaroon_file))
 }
+
+/// Sets the onion messaging feature bit (described in this PR:
+/// https://github.com/lightning/bolts/pull/759/), to signal that we support
+/// onion messaging. This needs to be done every time we start up, because LND
+/// does not currently persist the custom feature bits that are set via the RPC.
+async fn set_feature_bit(mut client: Client) {
```

Comment:
I'd like to be able to test these functions without needing to pass around a full `Client` - I think that'll get annoying / verbose to mock. Could we define a trait that only has the functionality we need + pass in here as a trait impl so that it's easier to mock?